docs/configuration/access_log.rst

```diff
@@ -21,71 +21,102 @@ Access logs are configured as part of the :ref:`HTTP connection manager config
+fields may have slightly different meanings, depending on what type of log it is.  Differences
+are noted.
+
 The following command operators are supported:
 
 %START_TIME%
-  Request start time including milliseconds.
+  HTTP
+    Request start time including milliseconds.
+
+  TCP
+    Downstream connection start time including milliseconds.
 
 %BYTES_RECEIVED%
-  Body bytes received.
+  HTTP
+    Body bytes received.
+
+  TCP
+    Downstream bytes received on connection.
 
 %PROTOCOL%
-  Protocol. Currently either *HTTP/1.1* or *HTTP/2*.
+  HTTP
+    Protocol. Currently either *HTTP/1.1* or *HTTP/2*.
+
+  TCP
+    Not implemented ("-").
 
 %RESPONSE_CODE%
-  HTTP response code. Note that a response code of '0' means that the server never sent the
-  beginning of a response. This generally means that the (downstream) client disconnected.
+  HTTP
+    HTTP response code. Note that a response code of '0' means that the server never sent the
+    beginning of a response. This generally means that the (downstream) client disconnected.
+
+  TCP
+    Not implemented ("-").
 
 %BYTES_SENT%
-  Body bytes sent.
+  HTTP
+    Body bytes sent.
+
+  TCP
+    Downstream bytes sent on connection.
 
 %DURATION%
-  Total duration in milliseconds of the request from the start time to the last byte out.
+  HTTP
+    Total duration in milliseconds of the request from the start time to the last byte out.
+
+  TCP
+    Total duration of the downstream connection.
 
 %RESPONSE_FLAGS%
-  Additional details about the response, if any. Possible values are:
-
-  * **LH**: Local service failed :ref:`health check request <arch_overview_health_checking>` in addition to 503 response code.
-  * **UH**: No healthy upstream hosts in upstream cluster in addition to 503 response code.
-  * **UT**: Upstream request timeout in addition to 504 response code.
-  * **LR**: Connection local reset in addition to 503 response code.
-  * **UR**: Upstream remote reset in addition to 503 response code.
-  * **UF**: Upstream connection failure in addition to 503 response code.
-  * **UC**: Upstream connection termination in addition to 503 response code.
-  * **UO**: Upstream overflow (:ref:`circuit breaking <arch_overview_circuit_break>`) in addition to 503 response code.
-  * **NR**: No :ref:`route configured <arch_overview_http_routing>` for a given request in addition to 404 response code.
-  * **DI**: The request processing was delayed for a period specified via :ref:`fault injection <config_http_filters_fault_injection>`.
-  * **FI**: The request was aborted with a response code specified via :ref:`fault injection <config_http_filters_fault_injection>`.
-  * **RL**: The request was ratelimited locally by the :ref:`HTTP rate limit filter <config_http_filters_rate_limit>` in addition to 429 response code.
+  Additional details about the response or connection, if any. For TCP connections, the specified response code
```

Comment:
nit: "For TCP connections, the specified response code does not apply" is a bit confusing. I think you mean to say something like all codes don't apply, etc.